lib/src/main/proto/bio/terra/tanagra/underlay/tables.proto

```diff
@@ -48,3 +48,18 @@ message ColumnId {
   // The name of the column.
   string column = 3;
 }
+
+// A filter specification to get a subset of the rows in a table.
+message TableFilter {
+  // A condition on a particular column
+  // TODO: allow multiple ColumnFilters and AND/OR'ing them together
```

Comment:
For future reference, mirroring the entity filter API here, but for tables rather than entities, could be a consistent way of doing this: https://github.com/DataBiosphere/tanagra/blob/328febf90e7f8a5c31b7ab8258d5a1bcbdb0bcb2/api/src/main/resources/api/service_openapi.yaml#L439

Reply:
duplicate comment, closing.